docs/source/crds.rst

```diff
@@ -0,0 +1,118 @@
+.. _custom-resources-used-by-the-test-operator:
+
+==========================================
+Custom Resources Used By the test-operator
+==========================================
+
+This file contains definitions of all custom resources that are accepted by
+the test-operator. Currently, there are two custom resouces:
+
+* :ref:`tempest-custom-resource`
+
+* :ref:`tobiko-custom-resource`
+
+
+.. _tempest-custom-resource:
+
+Tempest Custom Resource
+=======================
+
+.. literalinclude:: ../../config/samples/test_v1beta1_tempest.yaml
+   :language: yaml
+
+
+.. _custom-tempest-configuration:
+
+Custom Tempest Configuration
+----------------------------
+To configure tempest via tempest.conf use the :code:`tempestconfRun.overrides`
+parameter. This parameter accepts a list of key value pairs that specify values
+that should be written to tempest.conf generated inside the container.
+
+For example this definition of Tempest object:
+
+.. code-block:: yaml
+
+    ---
+    apiVersion: test.openstack.org/v1beta1
+    kind: Tempest
+    metadata:
+      name: tempest-tests
+      namespace: openstack
+    spec:
+      containerImage: quay.io/podified-antelope-centos9/openstack-tempest:current-podified
+      tempestRun:
+        includeList: | # <-- Use | to preserve \n
+          tempest.api.identity.v3.*
+        concurrency: 8
+      tempestconfRun:
+          overrides: |
+            auth.admin_username admin
+            auth.admin_password 1234
+
+will ensure that tempest will be executed with tempest.conf that looks like this:
```

Comment:
I think that this was already covered in https://openstack-k8s-operators.github.io/test-operator/guide.html#executing-tempest-tests
Let's avoid any duplication, it will confuse people.
I'd just link that from here and would include a sample of tempest configuration similarly as you did for tobiko on line 70.